src/prototype_room_keeper.js

```diff
@@ -120,7 +120,9 @@ Room.prototype.spawnKeepersEveryTicks = function(ticks) {
   let returnValue = false;
   if (Game.rooms[this.memory.base] && Game.rooms[this.memory.base].controller) {
     if (Game.rooms[this.memory.base].controller.level >= config.keepers.minControllerLevel) {
-      this.checkForWatcher();
+      // TODO I don't know what the watcher is good for, keeper rooms need to
```

Comment:
i did that part... they gain you acsess for the room, to see what is happening in that room
i send them first and the "anti" keepers and sources

Reply:
btw the move to the center of the room and stay there...
in my version i use them also to defend Invader (InvaderCore) neibours

Reply:
Ok, thx. I had some rooms which were constantly sending watcher creeps to one room, and keeper room harvesting is not enabled right now.

When coming to that topic I will have a second look.